src/simulators/state_chunk.hpp

```diff
@@ -890,6 +899,8 @@ void StateChunk<state_t>::apply_ops_multi_shots(InputIterator first, InputIterat
   int_t i;
   int_t i_begin,n_shots;
 
+  bool pauli_only = noise.pauli_only();
```

Comment:
I believe that `pauli_only` should be renamed with `batched_pauli_ops`.
Also, it is better to check `NoiseModel.opset()` here instead of adding a new method `pauli_only()`.

Reply:
removed